diff --git a/node_modules/@upstash/rag-chat/index.mjs b/node_modules/@upstash/rag-chat/index.mjs
index 81ba6b5..d8bf79b 100644
--- a/node_modules/@upstash/rag-chat/index.mjs
+++ b/node_modules/@upstash/rag-chat/index.mjs
@@ -1,3 +1,4 @@
+import fs from "fs"
 // src/error/model.ts
 var UpstashError = class extends Error {
   constructor(message) {
@@ -434,7 +435,7 @@ var FileDataLoader = class {
         apiKeyAuth: this.config.processor.options.apiKey
       }
     });
-    const fileData = await Bun.file(this.config.fileSource).text();
+    const fileData = fs.readFileSync(this.config.fileSource, 'utf-8');
     const response = await client.general.partition({
       //@ts-expect-error Will be fixed soon
       partitionParameters: {
